docs/release_process.md

```diff
@@ -3,6 +3,8 @@
 The schedule for the upcoming v0.24 release is:
 - August 20, 2025: Create release branch `release/0.24.x`.
 - **September 17, 2025: Release v0.24.0.**
+- September 24, 2025: Release v0.24.1 (if necessary).
+- October 1, 2025: Release v0.24.2 (if necessary).
```

Comment:
I'm not sure if these hard dates are needed - like what if we do a patch release after October 1? Anyway, just thinking out loud.

Reply:
Yeah, I'm not sure what's the best form here either. My goal with this was to let people know that they may expect patch releases to appear in the weeks following the main release. We usually do 1-2 patch releases short after the release so that's why I deliberately chose this number. But we can drop it if you find it too confusing.

Reply:
I think we should probably drop it but we can also just try it out and see how we like it. Not that big a deal.

Reply:
Let's leave it for one release to see how it goes.